hello_world/README.md

```diff
@@ -1,82 +1,32 @@
-### Hello World
-Welcome to the step-by-step guide of creating the Hello World application with Lisk Alpha SDK.
+## Hello World
+This is the reference implementation for the Hello World example, find the [full step by step guide here](https://github.com/LiskHQ/lisk-sdk-examples/tree/RachBLondon-patch-1/hello_world).
 
-The purpose of Hello World application is rather to explain the flow of using Alpha SDK Custom Transaction implementations. 
-The implementation is saving the string value of the "hello" transaction's asset property to the asset property of the sender's account.
+If you have gi cloned this repo - you can install the dependencies:
 
-If the account of an address "16313739661670634666L" is able to afford for a HelloWorld transaction (fee is set to 1 LSK by default), the new "hello" property appears into this account's asset field.
-Hello World transaction implements only the required functions from the BaseTransaction abstract interface.
-The overview of Custom Transaction implementation you can find in the dedicated section.
-So after sending a valid `{"type": 10, "senderId": "16313739661670634666L", ... "asset": { "hello": "world" } }` transaction, the sender's account changes from: `{ address: "16313739661670634666L", ..., asset: null }`, to `{ "address": "16313739661670634666L", ..., "asset": {"hello": "world"}} }`. 
-
-The Hello World implementation goes as following:
-
-#### TYPE
-
-Set the HelloWorld transaction TYPE to 10. Every time a transaction is received, it gets differentiated by the type. The first 10 types, from 0-9 is reserved for the default Lisk Network functions.
 ```
-	static get TYPE () {
-		return 10;
-	}
+npm i 
 ```
-#### applyAsset
```

Comment:
What happened with all of these contents? 
Did they get removed or moved to other files? :)

Reply:
The content is the same as the tutorials, so I think it makes sense to just link to them, we can avoid inconsistencies and reduce maintenance.